mobile/examples/phi-3/ios/LocalLLM/LocalLLM/lib/README.md

```diff
@@ -0,0 +1,2 @@
+**Note:**
+This folder contains the dynamic library for ORT and ORT GenAI that's required for the project.
```

Comment:
Same here. Would it be better to tell the user how to copy these files here so there's no mismatch?

Or if we want to checkin the headers and the dylibs, are the steps to checkout and build ORT and ORT-genai optional (so installing python and python-flatbuffers and cmake are also optional)?